starters/express-apollo-prisma/src/queue/job-generator-handler.spec.ts

```diff
@@ -0,0 +1,93 @@
+import amqplib from 'amqplib';
+import { createJobGeneratorHandler } from './job-generator-handler';
+
+const MOCK_URL = 'amqp://localhost:5673';
+
+const SPY_AMQPLIB_CONNECT = jest.spyOn(amqplib, 'connect');
+
+describe('.createJobGeneratorHandler', () => {
+	let req: any;
```

Comment:
Req and Res can be imported from src/mocks/express. Please check out example in src\healthcheck\healthcheck-handler.spec.ts and src\graphql\server-context\server-context-middleware-options.spec.ts